flow-typed/just-safe-get.js

```diff
@@ -0,0 +1,8 @@
+// @flow
```

Comment:
@angus-c - FYI, in case you want to add types to https://github.com/angus-c/just :P